src/main/java/com/jalasoft/webservice/model/ImageConvert.java

```diff
@@ -47,14 +48,19 @@ public Response convert(Criteria criteria) throws IOException {
             //Instantiating the PDFRenderer class
             PDFRenderer renderer = new PDFRenderer(document);
             int count = document.getNumberOfPages();
+            String [] filePaths = new String[count];
+
             for (int page = 0; page < count; ++page) {
 
```

Comment:
Normally there is not a space line after a statement.

Reply:
solved